app/views/homeTab/components/Home.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+import React from 'react'
+import { Container, Content, Text, Button, Form, Item, Icon, Input, List, ListItem, Body, Label, Drawer } from 'native-base';
+import Sidebar from './Sidebar'
+import getTheme from '../../../theme/native-base-theme/components';
+
+export default class Home extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      showForm: false
+    }
+    this.toggleForm = this.toggleForm.bind(this)
+    this.openSidebar = this.openSidebar.bind(this)
+    this.closeSidebar = this.closeSidebar.bind(this)
+  }
+  toggleForm() {
+    const formState = this.state.showForm
+    this.setState({
+      showForm: !formState
+    })
+  }
+  openSidebar() {
+    this.sidebar._root.open()
+  }
+  closeSidebar() {
+    this.sidebar._root.close()
+  }
+  render() {
+    const formState = this.state.showForm
+    return (
+      <Drawer
+        ref={ref => {this.sidebar = ref}}
+        content={<Sidebar navigator={this.navigator} />}
+        styles={{drawer: {backgroundColor: "white"}}}
+        onClose={this.closeSidebar} >
+        <Container>
+          <Content>
+            <Text style={getTheme()['NativeBase.Title']}>My Groups</Text>
```

Comment:
The Title component doesn't seem to render unless we wrap it inside a Header and Body.